src/dos/program_mount.cpp

```diff
@@ -462,20 +462,21 @@ void MOUNT::AddMessages() {
 	        "\n"
 	        "Notes:\n"
 	        "  - '-t overlay' redirects writes for mounted drive to another directory.\n"
+	        "  - '-usecd #' gives direct access to a CD-ROM drive (needed for CD audio).\n"
```

Comment:
`-usecd ID` would be better, then explain in a few words what ID is supposed to be.

Reply:
Is it broken on Mac?  I don't have one.  I do dual boot Windows but haven't tested there.  Anyway, that section is already `#ifdef`'d for platform specific notes so I can put a note about that.

Reply:
Based on what others have reported (I never tried it), it only works on Windows, so now we have Windows support but via the SDL mixer, not the DOSBox mixer, and with this feature Linux support via the DOSBox mixer, and that's it. No support on macOS.

Reply:
From what I've seen, SDL CD support doesn't use **any** mixer.  It doesn't actually read digital audio.  It just issues a command to the CD drive to tell it to start playing.  On Linux, this method is just broken on modern systems which is why I wrote this.  Maybe this method still works on Windows (this is how DOS games worked anyway) but there's no sound code here.  If anything gets mixed, it's up to the OS.

SDL does have code for Mac OSX, though no idea how well it works.  I'll test on Windows later.

Reply:
>  It just issues a command to the CD drive to tell it to start playing.

Right, it's even worse then than I thought 😅 

> SDL does have code for Mac OSX, though no idea how well it works. I'll test on Windows later.

Thanks for that. I have a rather low interest in this feature, and the guy who (re-)introduced it disappeared 🤷🏻

Reply:
Yeah, that SDL code is pretty janky.  It's ripped out of version 1 which is just unmaintained nowadays.  If I was doing it, I would have at least tried to use the sdl12-compat library (re-implementation of the SDL 1 API using SDL 2 as a backend):  https://github.com/libsdl-org/sdl12-compat

That one looks like it tries to do digital audio properly.  I also have a low interest in messing with library code for other platforms though.

Reply:
Fair enough. I wasn't involved, but I guess nobody was super intersted in this and went "yeah, whatever" 🤷🏻

Reply:
So can you change `-usecd` to this (just format it properly, and feel free to reword it better and/or more succinctly):
```
'-usecd ID' gives direct access to a CD-ROM drive. This is needed for CD audio (only supported on Windows and Linux). Run 'mount -cd' to find out the list of valid IDs.
```

Reply:
I changed it to the requested text.  Not sure how best to format it.  You gave me 3 lines of text there and now the help command is paging.

![mount](https://github.com/dosbox-staging/dosbox-staging/assets/8128047/cc32866b-16de-4362-8e0d-2dfd090bd457)